main/performance.cpp

```diff
@@ -167,6 +189,24 @@ double Performance::get_monitor(Monitor p_monitor) const {
 			return PhysicsServer3D::get_singleton()->get_process_info(PhysicsServer3D::INFO_ISLAND_COUNT);
 		case AUDIO_OUTPUT_LATENCY:
 			return AudioServer::get_singleton()->get_output_latency();
+		case NAVIGATION_ACTIVE_MAPS:
+			return NavigationServer3D::get_singleton_mut()->get_process_info(NavigationServer3D::INFO_ACTIVE_MAPS);
```

Comment:
```suggestion
			return NavigationServer3D::get_singleton()->get_process_info(NavigationServer3D::INFO_ACTIVE_MAPS);
```

and for the rest of these metrics.   We really don't need to use the "mutable" version of these.

Reply:
Changed to NavigationServer3D::get_singleton() and get_process_info() to const.